project_euler/problem_01/sol1.py

```diff
@@ -9,5 +9,21 @@
     raw_input          # Python 2
 except NameError:
     raw_input = input  # Python 3
-n = int(raw_input().strip())
-print(sum([e for e in range(3, n) if e % 3 == 0 or e % 5 == 0]))
+
+def solution(n):
+    """Returns the sum of all the multiples of 3 or 5 below n.
+    
+    >>> solution(3)
```

Comment:
Please consider adding __solution(-7)__ just to be sure.

Reply:
I will make it tomorrow.